nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -195,6 +195,14 @@ private TableIdentifier toIdentifier(EntriesResponse.Entry entry) {
     return TableIdentifier.of(elements.toArray(new String[elements.size()]));
   }
 
+  public IcebergTable table(TableIdentifier tableIdentifier) {
+    return (IcebergTable) fetchContent(tableIdentifier);
```

Comment:
before casting I wonder whether this should actually check whether this is actually a table and throw an error if not

Reply:
Instead of throwing exception, I have updated to returning null. So, that the behaviour will same as before adding the view changes. 

https://github.com/apache/iceberg/pull/8909/files#diff-4abd6d90c69587709bc754be9ea305080395bad9ac778176a0e3ab2563ac8464R198